test/d2l-localize-behavior.html

```diff
@@ -200,6 +208,18 @@
 					});
 
 					it('should format a time using specified format', function() {
+						var val = elem.formatTime(date, {format: 'full'});
+						expect(val).to.equal('5:13 PM Canada - Toronto');
+					});
+
+					it('should not fail if timezone data is missing', function() {
+						htmlElem.removeAttribute('data-timezone');
+						var val = elem.formatTime(date, {format: 'full'});
+						expect(val).to.equal('5:13 PM ');
```

Comment:
Not really related, but that extra space seems a bit weird, is that a bug in `formatTime`? ("bug" might be a strong word for an extra space, granted 😜)

Reply:
Yeah it's kind of a bug in that it has no timezone name so it just appends a space and then nothing. It's "existing" behaviour, but agreed that its wrong.